docs/en/quickstart.md

```diff
@@ -38,99 +39,102 @@ Ultralytics provides various installation methods including pip, conda, and Dock
         pip install git+https://github.com/ultralytics/ultralytics.git@main
         ```
 
+````
+=== "Conda install"
+
+    Conda is an alternative package manager to pip which may also be used for installation. Visit Anaconda for more details at [https://anaconda.org/conda-forge/ultralytics](https://anaconda.org/conda-forge/ultralytics). Ultralytics feedstock repository for updating the conda package is at [https://github.com/conda-forge/ultralytics-feedstock/](https://github.com/conda-forge/ultralytics-feedstock/).
```

Comment:
This is being processed incorrectly for some reason

Reply:
Nope, the original was correct, but there were two new lines which should have been one (manually fixed now)